lbr.py

```diff
@@ -347,17 +350,24 @@ def is_imix(t):
 def inc_pair(first, second='JCC', suffix='non-fusible'):
   c = '%s-%s %s' % (first, second, suffix)
   k = 'cond_%s' % c if second == 'JCC' else c
-  if k in glob: glob[k] += 1
-  else:
-    glob[k] = 1
-    if second == 'JCC':
-      global Insts_cond
-      Insts_cond += [c]
+  new = inc_stat(k)
+  if new and second == 'JCC': # new JCC paired stat added
+    global Insts_cond
+    Insts_cond += [c]
   if second == 'JCC' and suffix == 'non-fusible':
     glob['counted_non-fusible'] += 1
     return True
```

Comment:
note return value does not consider inc_stat() return value

Reply:
Why should it? The original code also didn't consider a new/existing stat in its return value, I kept the same functionality.